src/ctirs/api/v1/stix_files_v2/views.py

```diff
@@ -34,6 +35,7 @@ def get_api_stix_files_v2_sighting_count(request):
 
 # /api/v1/stix_files_v2/<observed_data_id>/sighting
 @csrf_exempt
+@api_key_auth
 def sighting(request, observed_data_id):
     ctirs_auth_user = authentication(request)
```

Comment:
You have not deleted authentication.

Reply:
```
ctirs_auth_user = authentication(request)
```

In `sighiting`, `ctirs_auth_user`  is required because it is used for other purposes besides authentication.

---

`ctirs_auth_user` なのですが、認証以外の別の用途で必要になるので消していません。
二重にチェック auth_user の値をとっているのは承知の上でこの実装としています。